maintenance/dumpRDF.php

```diff
@@ -176,10 +184,16 @@ private function runExport() {
 			$pages = explode( '|', $this->getOption( 'page' ) );
 		}
 
-		if ( $this->hasOption( 'server' ) ) {
+        if ( $this->hasOption( 'namespace' ) ) {
+            $namespaces = array_map('constant', explode( '|', $this->getOption( 'namespace' ) ) );
+            $restrictNamespaceTo = $namespaces;
```

Comment:
This line is not needed, just assign to the variable directly

Reply:
Thanks for the quick review! I've assigned the value directly now and also fixed a typo and missing quotes in the top comment section.